common/repeat_timer.go

```diff
@@ -36,7 +36,8 @@ func (t *defaultTicker) Chan() <-chan time.Time {
 
 // Implements Ticker
 func (t *defaultTicker) Stop() {
-	t.Stop()
+	tt := time.Ticker(*t)
```

Comment:
It should be `((*time.Ticker)(t)).Stop()`. I think that one-liner is more clear.  Also I'm not sure if the above is correct.  In general we shouldn't assume that things can be dereferenced like that... often times objects are meant to be passed only by reference.  For example, see the documentation for sync.Mutex.`A Mutex must not be copied after first use.`.  In this case, nothing says a Ticker can't be copied, but it might be that the behavior is just undefined.  Whereas the one-liner is definitely going to work.